controller/destination/server.go

```diff
@@ -126,6 +132,29 @@ func (s *server) Get(dest *common.Destination, stream pb.Destination_GetServer)
 	return err
 }
 
+func (s *server) echoIPDestination(host string, port int, stream pb.Destination_GetServer) error {
```

Comment:
as you don't need the server here, you can just declare the function standalone:
```golang
func echoIPDestination(host...
```